data/mods/MindOverMatter/items/teleporter_start_items.json

```diff
@@ -0,0 +1,67 @@
+[
+  {
+    "id": "mom_fusion_pistol",
+    "looks_like": "glock_17",
+    "type": "GUN",
+    "name": { "str": "Ch47 Fusion Pistol" },
+    "description": "The standard-issue officer's sidearm for the Imperial Navy, guaranteed manufactured on Tlalticpactli itself by scions of the Huey Calpulli, the Ch47 replaced the earlier Ch41 and is widely renowned for its accuracy and ease of maintenance.  This particular Ch47 even bears an eagle and snake on the grip, indicating it was produced in a factory in Anáhuac.  Hopefully it'll make a good melee weapon as well.",
+    "to_hit": -2,
+    "melee_damage": { "bash": 5 },
+    "reload_noise_volume": 5,
+    "weight": "800 g",
+    "volume": "1 L",
+    "longest_side": "250 mm",
+    "symbol": "(",
+    "color": "red",
+    "skill": "pistol",
+    "material": [ "superalloy", "steel" ],
+    "dispersion": 150,
+    "durability": 10,
+    "loudness": 20,
+    "reload": 200,
+    "range": 45,
+    "ammo_effects": [ "PLASMA", "PLASMA_BUBBLE", "INCENDIARY" ],
+    "ammo": [ "mom_fusion_ammo" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE_WELL", "item_restriction": [ "mom_fusion_mag" ] } ],
+    "flags": [ "NEVER_JAMS", "NON_FOULING", "NEEDS_NO_LUBE" ]
+  },
+  {
+    "id": "mom_fusion_mag",
+    "type": "MAGAZINE",
+    "name": { "str": "Ch47 power pack" },
+    "description": "A power pack for the Ch47.",
+    "weight": "40 g",
+    "volume": "20 ml",
+    "material": [ "superalloy" ],
+    "price": 10000,
+    "price_postapoc": 1,
+    "symbol": "#",
+    "color": "red",
+    "ammo_type": [ "mom_fusion_ammo" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "mom_fusion_ammo": 40 } } ]
+  },
+  {
+    "id": "mom_fusion_ammo",
+    "type": "AMMO",
+    "name": { "str_sp": "Ch47 power cartridges" },
+    "description": "Power cartridges for the Ch47. You had better make every shot count, because there's no way you're finding any more of these.",
```

Comment:
```suggestion
    "description": "Power cartridges for the Ch47.  You had better make every shot count, because there's no way you're finding any more of these.",
```